tests/integrationv2/fixtures.py

```diff
@@ -36,3 +47,42 @@ def _fn(provider_class: Provider, options: ProviderOptions, timeout=5):
         # Whether the processes succeeded or not, clean then up.
         for p in processes:
             p.join()
+
+
+def _swap_mtu(device, new_mtu):
+    """
+    Swap the device's current MTU for the requested MTU.
+    Return the original MTU so it can be reset later.
+    """
+    cmd = ["ifconfig", device]
```

Comment:
Technically `ifconfig` is supposed to be deprecated, but it's hit and miss, my Ubuntu Docker image didn't come with the replacement, so more of a nit.

You can also get this information by reading the file `/sys/class/net/<INTERFACE>/mtu`
```
root@6f4b3a9bd121:/# cat /sys/class/net/eth0/mtu
1500
```

Reply:
I think "ip" Is the portable way to do it. sysfs won't work on non-Linux

Reply:
I've made an update to use `ip link` to set and get the MTU. This is working on Ubuntu 18.04 and 16.04, which are the two platforms I've seen people using to run these tests.